src/Nox.Types/Types/HashedText/TypeOptions/HashedTextTypeOptions.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Nox.Types;
+
+public class HashedTextTypeOptions
+{
+    public HashingAlgorithm HashingAlgorithm { get; set; } = HashingAlgorithm.SHA256;
+    public int Salt { get; set; } = 64;
```

Comment:
Can we call this property "SaltLength" for clarity?

Reply:
Property name updated.